ASUWebPlatforms/webspark-ci
# WS2-1620: Fixing Content image overlap component

Ticket: https://asudev.jira.com/browse/WS2-1620

QA Steps:

1-Create a Content Image Overlay component.
2-Add an image using the Cropped Image Square (1:1).
3-Save the component and check how it looks. 

Please, repeat the same process but selecting  Cropped Image Wide (16:9) format instead(Step 2).

Both cases should the selected images should be displayed correctly.
